### PR TITLE
Added first version of the PHP client

### PR DESCRIPTION
**Pull request by tolvstein**

### CODE REVIEW

**Comment by twarberg** [comment]
Its considered best practice to omit `?>` on pure PHP files (http://www.php-fig.org/psr/psr-2/)

**Comment by twarberg** [comment]
Would personally prefer namespace `QuickPay\API`. @ta?

File: Quickpay/classes/Response.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    
+    
+    /**
+     * http_status
+     * 
+     * Returns the http_status code
+     * 
+     * @return int
+     */ 
+    public function http_status()
```
Comment (twarberg) [inline Quickpay/classes/Response.php]: Please don't mix camelCase and snake_case

**Comment by twarberg** [comment]
Generally looks good except for a few minor comments.

**Comment by ta** [comment]
+1 to `QuickPay\API` namespace - we should enforce that on all clients

File: Quickpay/Tests/classes/ExceptionTest.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Quickpay\Tests;
+
+use Quickpay\Classes\Exception;
```
Comment (ta) [inline Quickpay/Tests/classes/ExceptionTest.php]: Please use a `QuickPay\API\Exception` namespace - that is, replace "Classes" with "API" in all occurrences.

**Comment by ta** [comment]
@tolvstein: Nice to see that namespaces and tests have been added. Looks good apart from a few comments.

**Comment by tolvstein** [comment]
@ta @twarberg  - thanks for your feedback. I have pushed a new commit with your suggested changes.

**Comment by ta** [comment]
@tolvstein: Very cool! One question (nitpicking) - can we use QuickPay with a capital P?

**Comment by tolvstein** [comment]
@ta - done :)

**Comment by twarberg** [comment]
@tolvstein - Bør nok også være QuickPay som dir navn.

**Comment by tolvstein** [comment]
@twarberg - for some reason git didn't update the rename of the folder. This is now fixed.

**Pull request merged**